Description: Fix s390x stack smashing on lua error handler
 The current s390x jmpbuf size is too small, bump it to 64 bytes
 which is now plenty to save state when handling the jmpbuf jmps
Author: Colin Ian King <colin.king@ubuntu.com>
Last-Update: 2017-07-05

Index: zfs-linux-0.8.1/module/lua/ldo.c
===================================================================
--- zfs-linux-0.8.1.orig/module/lua/ldo.c
+++ zfs-linux-0.8.1/module/lua/ldo.c
@@ -61,7 +61,7 @@
 #elif defined(__mips__)
 #define JMP_BUF_CNT	12
 #elif defined(__s390x__)
-#define JMP_BUF_CNT	9
+#define JMP_BUF_CNT	64	/* maybe too large */
 #else
 #define	JMP_BUF_CNT	1
 #endif
